x-pack/plugins/reporting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -0,0 +1,212 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
+import puppeteer from 'puppeteer-core';
+import rimraf from 'rimraf';
+import * as Rx from 'rxjs';
+import { filter, map, mergeMap, partition, share } from 'rxjs/operators';
+import { safeChildProcess } from '../../safe_child_process';
+import { HeadlessChromiumDriver } from '../driver';
+import { args } from './args';
+
+const compactWhitespace = (str: string) => {
+  return str.replace(/\s+/, ' ');
+};
+
+interface IViewport {
+  width: number;
+  height: number;
+}
+
+interface IBrowserConfig {
+  disableSandbox: boolean;
+  proxy: any;
+}
+
+interface ILaunchArgs {
+  viewport: IViewport;
+  browserTimezone: string;
+}
+
+export class HeadlessChromiumDriverFactory {
+  public type = 'chromium';
+  private binaryPath: string;
+  private logger: any;
+  private browserConfig: IBrowserConfig;
+
+  constructor(binaryPath: string, logger: any, browserConfig: IBrowserConfig) {
+    this.binaryPath = binaryPath;
+    this.logger = logger.clone(['chromium-driver-factory']);
+    this.browserConfig = browserConfig;
+  }
+
+  public test({ viewport, browserTimezone }: ILaunchArgs, log: (message: string) => any) {
+    const userDataDir = fs.mkdtempSync(path.join(os.tmpdir(), 'chromium-'));
+    const chromiumArgs = args({
+      userDataDir,
+      viewport,
+      verboseLogging: true,
```

Comment:
This is what generates the `chrome_debug.log` file

Reply:
Looks like the file name can't be customized, sadly.